src/base/TilesRendererBase.js

```diff
@@ -551,7 +553,7 @@ export class TilesRendererBase {
 						}
 
 						const uri = parseTile.content.uri;
-						const extension = uri.split( /\./g ).pop();
+						const extension = getUrlExtension( uri ) || '';
```

Comment:
Is there a specific scenario where this fallback to an empty string is necessary or is it defensive programming? If it's the latter I think I'd prefer to keep it as `null` because I think it will be more clear that something went wrong.

Reply:
I had no specific scenario in mind. I'll switch this to `const extension = getUrlExtension( uri );`. My only goal here was to preserve the original behavior where `extension` is assigned a string so long as `uri` is a string.